packages/manager/apps/veeam-backup/src/pages/dashboard/OfferProgress.component.tsx

```diff
@@ -1,53 +1,120 @@
-import React from 'react';
+import { Fragment, useMemo } from 'react';
 import {
   OsdsIcon,
   OsdsProgressBar,
   OsdsText,
   OsdsTooltip,
   OsdsTooltipContent,
+  OsdsSkeleton,
+  OsdsChip,
 } from '@ovhcloud/ods-components/react';
+
+import { VeeamBackupOffer } from '@ovh-ux/manager-module-vcd-api';
+
 import { Description } from '@ovh-ux/manager-react-components';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
 import {
   ODS_ICON_NAME,
   ODS_ICON_SIZE,
   ODS_TEXT_SIZE,
   ODS_TEXT_LEVEL,
+  ODS_SKELETON_SIZE,
+  ODS_CHIP_SIZE,
 } from '@ovhcloud/ods-components';
 import { useTranslation } from 'react-i18next';
-import { VeeamBackupOffer } from '@ovh-ux/manager-module-vcd-api';
 
-export const OfferProgress: React.FC<VeeamBackupOffer> = ({
-  name,
-  quotaInTB,
-  usedSpaceInGB,
-}) => {
+import useVeeamBackupConsumption from '@/data/hooks/useVeeamBackupConsumption';
+
+import { getRealOfferConsumption } from './helpers';
+
+export const OfferProgress = ({
+  offer,
+  id,
+}: {
+  offer: VeeamBackupOffer;
+  id: string;
+}): JSX.Element => {
   const { t } = useTranslation('dashboard');
-  const percent = (usedSpaceInGB / (quotaInTB * 10)).toFixed(2);
+
+  const {
+    data: consumptions,
+    isLoading: isLoadingConsumption,
+  } = useVeeamBackupConsumption(id);
+
+  const {
+    usedSpaceInGB,
+    quotaInTB,
+    percent,
+    name,
+    protectionPrimaryRegion,
+    protectionReplicatedRegion,
+  } = useMemo(() => {
+    const realOffer = getRealOfferConsumption(offer, consumptions);
+    return {
+      ...realOffer,
+      percent: (realOffer.usedSpaceInGB / (realOffer.quotaInTB * 10)).toFixed(
+        2,
+      ),
+    };
+  }, [consumptions, offer]);
 
   return (
-    <div className="flex flex-col">
-      <div className="flex mb-3">
-        <OsdsText
-          level={ODS_TEXT_LEVEL.body}
-          size={ODS_TEXT_SIZE._500}
-          color={ODS_THEME_COLOR_INTENT.text}
-        >
-          {t('space_summary', { usedSpaceInGB, quotaInTB, percent })}
-        </OsdsText>
-        <OsdsTooltip>
-          <OsdsIcon
-            className="ml-4"
-            name={ODS_ICON_NAME.HELP_CIRCLE}
-            color={ODS_THEME_COLOR_INTENT.text}
-            size={ODS_ICON_SIZE.xs}
-          />
-          <OsdsTooltipContent slot="tooltip-content">
-            <Description>{t(`${name}_tooltip`)}</Description>
-          </OsdsTooltipContent>
-        </OsdsTooltip>
-      </div>
-      <OsdsProgressBar max={quotaInTB * 1000} value={usedSpaceInGB} />
-    </div>
+    <Fragment>
+      {isLoadingConsumption ? (
+        <>
+          <OsdsSkeleton inline size={ODS_SKELETON_SIZE.sm} />
+          <OsdsSkeleton size={ODS_SKELETON_SIZE.md} />
+        </>
+      ) : (
```

Comment:
Not blocker but It's more readable with :

```suggestion
    <Fragment>
      if (isLoadingConsumption) {
      return (
          <OsdsSkeleton inline size={ODS_SKELETON_SIZE.sm} />
          <OsdsSkeleton size={ODS_SKELETON_SIZE.md} />
      )
```